internal/agentdeployer/info.go

```diff
@@ -13,6 +13,27 @@ const (
 	elasticAgentTagsEnv = "ELASTIC_AGENT_TAGS"
 )
 
+type AgentSettings struct {
```

Comment:
Moved to an independent struct in order to be used as embbeded struct here and in the test config. That makes possible to copy all fields at one in https://github.com/elastic/elastic-package/pull/1822/files#diff-de89bc59d9fa58f6220a67bda14afddc8f04ef1b816e81c2ca851711625f5834R339